crates/sources/src/indirect.rs

```diff
@@ -498,6 +498,10 @@ fn indirect_materialization(
             resources,
             threshold,
         ),
+        // Dekaf isn't a pluggable connector, and so does not have dynamic config.
```

Comment:
It could be, though, especially if config grows to be large. There's no reason not to treat it like any other config that I'm aware of. In practice, it will remain inline unless it gets large.

Reply:
Updated